syntax = "proto3";

import "mlagents/envs/communicator_objects/unity_rl_input.proto";
import "mlagents/envs/communicator_objects/unity_rl_initialization_input.proto";

option csharp_namespace = "MLAgents.CommunicatorObjects";
package communicator_objects;

message UnityInputProto {
    UnityRLInputProto rl_input = 1;
    UnityRLInitializationInputProto rl_initialization_input = 2;

    //More messages can be added here
}
